src/spawn_sync.cc

```diff
@@ -814,9 +814,7 @@ int SyncProcessRunner::ParseStdioOptions(Local<Value> js_value) {
   HandleScope scope(env()->isolate());
   Local<Array> js_stdio_options;
 
-  if (!js_value->IsArray())
-    return UV_EINVAL;
-
+  CHECK(js_value->IsArray());
```

Comment:
Yes, before this PR, `js_value` might be not an array since it is not sanitized in JS. After this PR, they should be sanitized and it would be a bug if it is not an array here.